GPU/GPUTracking/TPCClusterFinder/TPCCFCalibration.h

```diff
@@ -42,11 +44,20 @@ struct TPCPadGainCorrectionStepNum<unsigned short> {
 
 struct TPCCFCalibration {
  public:
+  // Calibration parameters for cluster finder
+  float qmaxCutoff = 3.f;
+  float qtotCutoff = 0.f;
+  float extendInnerChargeThreshold = 0.f;
+  float minSplitChargeNum = 1.f;
+  float noiseSuppressionEpsilon = 10.f;
```

Comment:
I don't quite understand why you have copies of these values here. Cannot you use directly the values from the GPUParam.rec?

Reply:
Totally missed that the GPUSettingsRec struct is also available on GPU. Which makes a lot of sense of course. Parameters are now accessed from there directly instead.